tests/features/comments/add_comment.feature

```diff
@@ -118,7 +72,7 @@ Feature: Add comments
       | discussion   | Is gossip bad?      | validated |
       | document     | Wikileaks           | validated |
 
-  Scenario Outline: Comments are disallowed for anonymous users in closed collections.
+  Scenario Outline: Comments are disallowed for anonymous users.
```

Comment:
It looks like it was the intention to add test coverage here to ensure that anonymous users can now not comment in both open and closed collections. The test was not changed though, so we now have no coverage any more for anonymous users commenting in open collections.